lib/oli_web/live/certificates/components/design_tab.ex

```diff
@@ -276,10 +280,20 @@ defmodule OliWeb.Certificates.Components.DesignTab do
   end
 
   def handle_event("cancel", %{"ref" => ref}, socket) do
-    {:noreply,
-     socket
-     |> cancel_upload(:logo, ref)
-     |> assign(logo_upload_errors: formatted_upload_errors(socket.assigns.uploads.logo, ref))}
+    socket = cancel_upload(socket, :logo, ref)
+
+    uploads = socket.assigns.uploads.logo
+
+    updated_uploads = %{
```

Comment:
I think we can remove this part altogether and in the `formatted_upload_errors/2` function at the end just do:

```elixir 
if uploads.entries > uploads.max_entries, do:
["You can upload up to #{uploads.max_entries} files only." | errors],
else: errors
```

Reply:
Nice catch!